.github/workflows/failing-weekly.yaml

```diff
@@ -0,0 +1,60 @@
+name: failing-weekly
+
+on:
+  workflow_dispatch:
+  schedule:
+    # at 3am weekly on Sunday
+    - cron: '0 3 * * 0'
+
```

Comment:
Would it be sensible to commit a change to run on PR push so we can see the test fail, and then revert that commit? That would provide feedback in the PR of the expected outcome.

Reply:
So that it runs on any `dev` version of `prisma/prisma`?

That is currently how it is implemented in our main workflow - but the "failing" tests we _know_ do not succeed because of a - in this case - limitation of the database. So this test is tracking if an update of the database surprisingly fixes that limitation.

Reply:
We already know it fails and why and want to move it away but keep it around to know if maybe one day it succeeds.